server/build/babel/preset.js

```diff
@@ -7,10 +7,21 @@ const productionPlugins = isProduction ? [
   require.resolve('babel-plugin-transform-react-remove-prop-types')
 ] : []
 
+const es2015Config = {}
+
+// Do not transform ES2015 modules if we are inside Next.js
+// That's because webpack2 knows how to handle it.
+// (And that's how it can do code splitting)
+//
+// But in other environements like Jest, we should transform it.
+if (process.env.INSIDE_NEXT) {
```

Comment:
@nkzawa how about this:

* We allow user to provide some options to our babel config (to transform modules or not)
* If don't provide ^^^ option and runs inside Next.js we won't transform ES2015 modules?